packages/client/lib/util/parse.ts

```diff
@@ -246,6 +246,7 @@ async function parseGethParams(json: any) {
       block: null,
     })
   }
+  params.td = terminalTotalDifficulty
```

Comment:
you don't need to add this because it's just a few lines up that pushes `td` into the hardfork array for `merge`

Reply:
Interesting, you mean this line https://github.com/ethereumjs/ethereumjs-monorepo/blob/merge-kiln/packages/client/lib/util/parse.ts#L240-L246 ?

At first, I thought that we don't need it because of that line, but since the block is being defined as `null` and we were not passing the `totalTerminalDifficulty`; which was the reason for the hard fork being set to `london` and not to `merge`

Probably I am missing something? :thinking: